### PR TITLE
Add MarginPool interface

### PR DESCRIPTION
**Pull request by antoncoding**

### CODE REVIEW

File: contracts/interfaces/MarginPoolInterface.sol [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.6.10;
+
+interface MarginPoolInterface {
+    function transferToPool(
+        address asset,
+        address user,
+        uint256 amount
+    ) external;
```
Comment (aparnakr) [inline contracts/interfaces/MarginPoolInterface.sol:9]: `function transferToPool(
        address _asset,
        address _user,
        uint256 _amount
    ) external;`

File: contracts/interfaces/MarginPoolInterface.sol [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    function transferToUser(
+        address asset,
+        address user,
+        uint256 amount
+    ) external;
```
Comment (aparnakr) [inline contracts/interfaces/MarginPoolInterface.sol:15]: `function transferToUser(
        address _asset,
        address _user,
        uint256 _amount
    ) external`

File: contracts/interfaces/MarginPoolInterface.sol [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+    function transferToPool(
+        address[] memory asset,
+        address[] memory user,
+        uint256[] memory amount
+    ) external;
```
Comment (aparnakr) [inline contracts/interfaces/MarginPoolInterface.sol:21]: `function transferToPool(
        address[] memory _asset,
        address[] memory _user,
        uint256[] memory _amount
    ) external;`

File: contracts/interfaces/MarginPoolInterface.sol [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+    function transferToUser(
+        address[] memory asset,
+        address[] memory user,
+        uint256[] memory amount
+    ) external;
```
Comment (aparnakr) [inline contracts/interfaces/MarginPoolInterface.sol:27]: `function transferToUser(
        address[] memory _asset,
        address[] memory _user,
        uint256[] memory _amount
    ) external;`

**Comment by haythem96** [comment]
> Fix the naming of function arguments. All arguments should have _name right? @haythem96 thoughts?

Yeah agree.

**Pull request merged**